department-of-veterans-affairs/caseflow
# User Management Page - User Search Improvement

Resolves #14089 

### Description
Updates our User Management page to search based on user name or css id, dynamically.

### Acceptance Criteria
- [ ] Users with access to the user status management page have the ability to search for a user by name and CSS ID

### Testing Plan
1. Go to [`test/users`](http://localhost:3000/test/users) and sign in as the `SystemAdmin` user
1. Navigate to [User Management](http://localhost:3000/user_management)
 - [ ] Search for "`Jolly`" to confirm CSS search
 - [ ] Search for "`Wilson`" to confirm Name search
 - [ ] Search for "`Intake`" to confirm you can get a mix of CSS ID & Name based results
 - [ ] Confirm marking the user inactive still functions.
 - [ ] Confirm marking the user active still functions.

### User Facing Changes
 BEFORE|AFTER
 ---|---
![image](https://user-images.githubusercontent.com/6129479/81444075-8b5aaa80-9144-11ea-8a44-f3214331cfe9.png) | ![image](https://user-images.githubusercontent.com/6129479/81443992-649c7400-9144-11ea-9a2f-a3c99c81302a.png)
![image](https://user-images.githubusercontent.com/6129479/81444103-96add600-9144-11ea-83d5-94826d4fc927.png) | ![image](https://user-images.githubusercontent.com/6129479/81444033-77af4400-9144-11ea-80d3-b5942d621514.png)
![user_page_before](https://user-images.githubusercontent.com/6129479/81443678-daeca680-9143-11ea-8de0-9df6cf94304e.gif) | ![user_page_after](https://user-images.githubusercontent.com/6129479/81443685-dcb66a00-9143-11ea-82a4-f85d1e9aff2d.gif)
